R_functions/fetch_dwg.R

```diff
@@ -6,8 +6,9 @@ fetch_dwg <- function(fishery_name, ...){
     interview = "https://data.wa.gov/resource/rpax-ahqm.csv",
     catch = "https://data.wa.gov/resource/6y4e-8ftk.csv",
     water_bodies = "https://data.wa.gov/resource/nbd2-vdmz.csv",
-    closures = "https://data.wa.gov/resource/6zm6-iep6.csv"
+    closures = "https://data.wa.gov/resource/6zm6-iep6.csv",
     #,gear = "https://data.wa.gov/resource/d2ks-afhz.csv" #currently unused?
+    fishery_manager = "https://data.wa.gov/resource/vkjc-s5u8.csv"
```

Comment:
I am up for referring to this table as its specific name, rather than the thing it derived from. Maybe once Jess has updated FishApps and DWG I can go through the few places "fishery_manager" is referenced  in the ETL and update.